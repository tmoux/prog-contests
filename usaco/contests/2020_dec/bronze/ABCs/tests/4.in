692883429 431153715 238195703 192958012 454687726 885841441 647645738
