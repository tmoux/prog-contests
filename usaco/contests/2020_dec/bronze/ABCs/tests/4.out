192958012 238195703 454687726
